git_workflow.md

```diff
@@ -2,7 +2,24 @@
 TBD.
 
 ## Branch Strategy
-TBD.
+### Workflow
+study case: master branch for production, development for staging.
+origin/master -> local/master
+origin/development -> local/development (created from master branch)
+origin/feature or origin/bugfix -> lcoal/feature or local/bugfix (created from development branch)
+* USE development as DEFAULT branch 
+* DO NOT merge request / force push directly from local or origin feature/bugfix to origin master
+* create merge request to master branch ONLY FROM origin development branch
+* creeate merge request task to development branch ONLY
+* in case if hotfix needed, create branch hotfix/TaskName and merge it to master and development
```

Comment:
JIka butuh hotfix, apakah base branch nya boleh langsung dari branch `master`? Atau base branchnya tetap harus dari branch `development`, seperti yang dijelaskan di poin 1?

Reply:
kalau pendapat saya pribadi misal hotfixnya case di production gpp narik hotfix dari master, point pentingnya di merge ke development dan master. #cmiiw
cc: @adrianpdm

Reply:
Oke kang. Nanti boleh ditambahkan di poin di atas kang, misalnya jadi begini

```suggestion
* in case a production hotfix is needed, create branch `hotfix/TaskName` from branch `master`  and merge it to `master` and `development`.
```